src/ethereum-providers/connectors.js

```diff
@@ -34,6 +34,24 @@ export const connectors = [
         },
       }
     : null,
+  {
+    id: 'walletconnect',
+    properties: {
+      chainId: [1, 4, 137, 80001, 1666600000, 1666700000, 97, 56, 588, 1088], // add here to handle more injected chains
+      rpc: {
+        '1': 'https://mainnet.infura.io/v3/dd908cbc08714031bbf381d443083123',
+        '4': 'https://rinkey.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
```

Comment:
Please switch this to the Aragon RPC: https://rinkeby.eth.aragon.network

Reply:
Will do so too